util/file_test.go

```diff
@@ -25,8 +27,11 @@ func TestNewFile(t *testing.T) {
 
 	fileInfo, err := os.Stat(filename)
 	assert.NoError(t, err)
-	assert.True(t, 0600 == fileInfo.Mode().Perm())
 	assert.False(t, fileInfo.IsDir())
+
+	if runtime.GOOS != "windows" {
+		assert.True(t, 0600 == fileInfo.Mode().Perm())
```

Comment:
does windows have no concept of this or is the value different. if the latter we should check it